examples/logging/README.md

```diff
@@ -0,0 +1,340 @@
+- **Model accuracy:** what is the accuracy of the predictions vs human-labeled?
+
+DeepSparse Logging is designed to provide maximum flexibility for users to extract whatever data is needed from a production inference pipeline into the logging system of their choice. 
+
+## Metrics 
+DeepSparse Logging provides access to two types of metrics:
+- **System Logging Metrics** give operations teams access to granual performance metrics, diagnosing and isolating deployment system health. Examples include CPU utilization and query latency.
+     - See [below - UPDATE LINK](system-logging-metrics) for a full list of available metrics.
+
+- **Data Logging Metrics** give ML teams access to data at each stage of an ML pipeline, supporting downsteam tasks like measuring accuracy and data drift. Examples include raw inputs and projections thereof such as mean pixel value.
+     - See [below - UPDATE LINK](data-logging-functions) for a full list of built-in functions and how to create a custom function
+
+## Configuration
+DeepSparse Logging is configured via YAML files.
+
+**System Logging** is *enabled* by default and all metrics are pre-defined. It can be disabled 
+globally or at the group level by adding the key-value pairs with `on` or `off` status.
+        
+**Data Logging** is *disabled* by default. A YAML configuration file is used to specify which data or functions thereof to log. 
+There are 4 `stages` in the inference pipeline where Data Logging can occur:
+
+````
+stages: pipeline_inputs >> engine_inputs >> engine_outputs >> pipeline_outputs
+````
+
+The following YAML format is used to specify the configuration:
+```yaml
+loggers:
+     - prometheus
+     - custom_logger
+
+# system_logging: on/off    # flag to turn all system logging on/off; defaults to on
+system_logging:
+    group: on/off           # flag to turn off a group; if omitted, defaults to on
+    
+data_logging:
+    stage:
+      mapping:
+        # first function
+        - func: builtins:foo            # [REQUIRED STR] function identifier  (built-in or path to custom)
+          frequency: 1000               # [OPTIONAL INT] logging frequency    (default: 1000 - logs once per 1000 predictions)
+          target: all                   # [OPTIONAL STR] logger               (default: all)
+        # second function
+        - func: path/to/custom.py:bar  
+          frequency: 10000
+          target: prometheus
+        # ... list of as many functions as desired
+```
+
+<details>
+     <summary>Click for a tangible example YAML snippit</summary>
+
+```yaml
+loggers:
+    - prometheus
+    - custom_logger
+
+system_logging:
+    deployment_details: off
+    request_details: off
+    prediction_latency: on 
+    dynamic_batch_latency: off
+    # resource_utilization: on      << omitted groups are turned on by default
+    
+data_logging:
+     pipeline_inputs:
+       mapping:
+         - func: builtins/max                        # built-in function
```

Comment:
We need to remove `builtins` prefix and add a function name that actually exists (`max` doesn't):
https://github.com/neuralmagic/deepsparse/blob/main/src/deepsparse/loggers/metric_functions/built_ins.py